LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Brian Ford
+Copyright (c) 2019 Brian Ford
```

Comment:
I've seen the most common suggested change of this type as: 2014-2019. The year doesn't need to be updated at all, though; it doesn't affect the copyright.

Reply:
👍